RepoDb/RepoDb/OrderField.cs

```diff
@@ -140,7 +140,7 @@ public static IEnumerable<OrderField> Parse(object obj)
         /// <returns>The hashcode value.</returns>
         public override int GetHashCode()
         {
-            return Name.GetHashCode() + Order.GetHashCode();
+            return Name.GetHashCode() + (int)Order;
```

Comment:
@calumchisholm - this would definitely improve the performance. Same type casting is much really faster when compared to calling that 'GetHashCode()' method. Thanks for this PR.

Reply:
Happy to help - hope it makes a difference.